src/main/java/io/r2dbc/postgresql/message/frontend/CopyData.java

```diff
@@ -77,6 +78,16 @@ public boolean equals(Object o) {
         return Objects.equals(this.data, copyData.data);
     }
 
+    @Override
+    protected void deallocate() {
+        this.data.release();
+    }
+
+    @Override
+    public CopyData touch(Object hint) {
+        return this;
```

Comment:
Is this correct? A pass-through?

Reply:
`ReferenceCounted` API is made for fluent usage (`copyData.touch("foo").hashCode()`), see also https://github.com/netty/netty/blob/4.1/buffer/src/main/java/io/netty/buffer/AbstractReferenceCountedByteBuf.java#L88-L97.